microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -160,7 +160,7 @@ To drive WebView2 with Selenium and Microsoft Edge Driver:
 
     //Set the BinaryLocation to the filepath of the WebView2API Sample runtime
     edgeOptions.BinaryLocation = @"C:\path\to\your\webview2\project.exe";
-    EdgeDriver edgeDriver = new EdgeDriver(edgeOptions);
+    EdgeDriver e = new EdgeDriver(edgeOptions);
```

Comment:
A best practice in dev docs is not to use the var name that matches the class name. For beginners it can be difficult to understand. I suggest we update edgeOptions to something similar. What do you think?

Reply:
I see. Sounds good to me.

Reply:
Note: Two changes to variable names are discussed here:
edgeDriver
edgeOptions